### ISSUE TITLE
Add a link to Gosling editor from landing page

### ISSUE THREAD

**Issue by huyen-nguyen** [gosling-lang/gosling.js#1087]
I like the design of the current page, it would also be very useful to have a direct link to the editor, from the main website of [Gosling](http://gosling-lang.org/).

### Potential solution

Adding an item on nav bar as `Editor`. 

While we're at it, I'd suggest sorting the nav bar items alphabetically. It makes sense to me semantically, though it might look different to you.

    Tutorials • Themes • Examples • Editor • Docs • About
or

    About • Docs • Editor  • Examples • Themes • Tutorials

<img width="1746" alt="Screenshot 2024-07-25 at 6 52 25 PM" src="https://github.com/user-attachments/assets/28f5c28b-61f4-48a3-ab62-e79edd17e1d0">

**Comment by dvdkouril** [comment]
I agree with the link to the Editor from [gosling-lang.org](https://gosling-lang.org). 

@huyen-nguyen Would you like to make the PR? This belongs to https://github.com/gosling-lang/gosling-website if I'm not mistaken. I suggest to close this issue here.
